cfgov/v1/models/learn_page.py

```diff
@@ -104,6 +104,8 @@ def meta_image(self):
 class LearnPage(AbstractFilterPage):
     content = StreamField([
         ('full_width_text', organisms.FullWidthText()),
+        ('info_unit_group', organisms.InfoUnitGroup()),
+        # TODO: InfoUnitGroup2575Only is deprecated and will be removed.
```

Comment:
Oh maybe ignore my earlier comment, but could we elaborate here on what will need to happen to remove this? Unless it's coming in like the very next PR.

Reply:
Yes, my intention is to open a subsequent PR that essentially just removes the definition of and any code references to InfoUnitGroup2575Only. That will generate another schema migration for LearnPage but won't require a data migration since we're taking care of that in this RP.